src/EditorFeatures/Core/Implementation/Peek/PeekableItemSource.cs

```diff
@@ -40,6 +40,11 @@ internal sealed class PeekableItemSource : IPeekableItemSource
 
         public void AugmentPeekSession(IPeekSession session, IList<IPeekableItem> peekableItems)
         {
+            if (session.RelationshipName != PredefinedPeekRelationships.Definitions.Name)
```

Comment:
I'd probably use OrdinalIgnoreCase comparison here. Otherwise everything looks good to me.